cache_images/fedora_packaging.sh

```diff
@@ -94,6 +94,7 @@ INSTALL_PACKAGES=(\
     openssl-devel
     ostree-devel
     pandoc
+    perl-FindBin
```

Comment:
This is probably going to fail on pre-f33

Reply:
good catch